### PR TITLE
Add Rainbow Browser Extension to knownHosts

### PR DESCRIPTION
**Pull request by derHowie**
## Description
I'd like to add Rainbow's extension ids to the knownHosts array so that the permissions copy looks a little nicer in the Trezor popup.

### CODE REVIEW

File: packages/connect/src/data/config.ts [hunk 1]
```diff
@@ -36,6 +36,16 @@ export const config = {
             label: 'MetaMask',
             icon: '',
         },
+        {
+            origin: 'bpcdaglidgnlggelgbjfagekoapjmccp',
+            label: 'Rainbow',
+            icon: '',
+        },
```
Comment (karliatto) [inline packages/connect/src/data/config.ts]: I see the other one is the id for the rainbow extension in chromewebstore, but this one what is for other extension store?

Comment (karliatto) [reply packages/connect/src/data/config.ts]: Ok, it is used for Chrome store dev build.

**Review by karliatto — APPROVED** [review]

File: packages/connect/src/data/config.ts (hunk 1 above)
Comment (mroz22) [reply packages/connect/src/data/config.ts]: maybe lets use a different label for dev build, something like "Rainbow DEV"

**Review by mroz22 — CHANGES_REQUESTED** [review]

File: packages/connect/src/data/config.ts (hunk 1 above)
Comment (derHowie) [reply packages/connect/src/data/config.ts]: Sounds good, done

**Review by mroz22 — APPROVED** [review]

**Comment by karliatto** [comment]
@derHowie commits names are not passing our commit naming standars tests [commit-message-check](https://github.com/trezor/trezor-suite/actions/runs/8374971206/job/22967966664?pr=11625#logs)

Please have a look at https://www.conventionalcommits.org/

**Comment by derHowie** [comment]
> @derHowie commits names are not passing our commit naming standars tests [commit-message-check](https://github.com/trezor/trezor-suite/actions/runs/8374971206/job/22967966664?pr=11625#logs)
> 
> Please have a look at https://www.conventionalcommits.org/

Apologies @karliatto , I have amended the commit messages.

**Comment by karliatto** [comment]
> Apologies @karliatto , I have amended the commit messages.

@derHowie sorry but it looks like commit `Merge branch 'develop' into patch-1` is not allowed.

https://github.com/trezor/trezor-suite/actions/runs/8420395614/job/23057886161?pr=11625

**Comment by derHowie** [comment]
> > Apologies @karliatto , I have amended the commit messages.
> 
> @derHowie sorry but it looks like commit `Merge branch 'develop' into patch-1` is not allowed.
> 
> https://github.com/trezor/trezor-suite/actions/runs/8420395614/job/23057886161?pr=11625

Sorry again, I shouldn't have clicked the update branch button 😭

**Pull request merged**